main.go

```diff
@@ -31,7 +31,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/healthz"
 	"sigs.k8s.io/controller-runtime/pkg/log/zap"
 
-	serverlessv1alpha1 "github.com/davidesalerno/kogito-serverless-operator/api/v1alpha1"
+	serverlessv08 "github.com/davidesalerno/kogito-serverless-operator/api/v08"
```

Comment:
same.
or we can do the opposite, use serverlessv08 on other imports as well.
btw, is there a nickname for serverless, maybe srvlss?